core/src/main/java/amino/run/compiler/PolicyStub.java

```diff
@@ -299,12 +300,13 @@ public String getMethodContent(MethodStub m, boolean isDMMethod) {
      * @param isDMMethod
      * @return Stub code for RPC call based on method type.
      */
-    public String getMethodRPCContent(boolean isDMMethod) {
-        if (isDMMethod == true) {
-            return "$__result = $__makeKernelDMRPC($__method, $__params);";
-        } else {
-            return "$__result = $__makeKernelRPC($__method, $__params);";
+    public String getMethodRPCContent(MethodStub m, boolean isDMMethod) {
+        if (m.retType.equals(Void.TYPE)) {
+            if (isDMMethod) return "$__makeKernelDMRPC($__method, $__params);";
```

Comment:
suggestion: can we write if else condition like below for better readability.

if (isDMMethod) {
 return "$__result = $__makeKernelDMRPC($__method, $__params);";
}
else {
        return "$__result = $__makeKernelRPC($__method, $__params);";
}

Reply:
Added curly braces for better readability.
IMHO, ''else" statement is not required as we return from if statement itself.